wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -321,11 +321,11 @@ data Account = Account
 deriveJSON Serokell.defaultOptions ''Account
 
 instance Arbitrary Account where
-  arbitrary = Account . fromString <$> elements ["DEADBeef", "123456"]
-                                   <*> listOf1 arbitrary
-                                   <*> arbitrary
-                                   <*> pure "My account"
-                                   <*> arbitrary
+  arbitrary = Account <$> arbitrary
+                      <*> listOf1 arbitrary
```

Comment:
Probably it would be better to relax that `listOf1` from the `Arbitrary` instance. I know I was the guy which wrote this code, and that I probably did it in order for our Swagger spec to give back something nice rather than an empty list, but when we start getting serious about testing we should ask yourselves if your QC `Arbitrary` instances are modelling the domain correctly, and definitely an `Account` can have 0 or more addresses associated to it 😉